skimage/morphology/tests/test_ccomp.py

```diff
@@ -103,11 +113,17 @@ def test_background_one_region_center(self):
                       [0, 1, 0],
                       [0, 0, 0]])
 
-        assert_array_equal(label(x, neighbors=4, background=0),
+        with expected_warnings(["use 'connectivity'"]):
+            assert_array_equal(label(x, neighbors=4, background=0),
+                               [[0, 0, 0],
+                                [0, 1, 0],
+                                [0, 0, 0]])
```

Comment:
Why not use connectivity directly here, like you've done with other tests?

Reply:
I mostly just duplicated the tests to ensure that both the old and new behaviour give the expected results during the deprecation cycle